src/uwtools/exceptions.py

```diff
@@ -2,8 +2,16 @@
 Custom exceptions for the UWTools package.
 """
 
+from abc import ABC
 
-class UWConfigError(Exception):
+
+class UWError(ABC, Exception):
```

Comment:
Why do we need this to be an abstract class?

Reply:
This was the offline discussion suggestion that was included here - to make a parent general UWError class, of which ConfigErrors are one type, with additional ones to be included as needed

Reply:
I think that inheritance alone gives you that for these simple Exception classes and you wouldn't need it to also be abstract.

Reply:
@maddenp-noaa If this works for what you were thinking, I'll revert back no prob

Reply:
I think it comes down to the question: Do we want to be able to raise a `UWError`, or force ourselves to raise something more specific than that. I think I'd err on the side of removing `ABC` from the parent classes and accepting that we might raise a fairly generic `UWError if a more-specific exception isn't obvious.

The other half of the original question was: Do we now have a use case for a new _more-specific_ error? Are we raising `UWConfigError` where the issue isn't actually a *configuration* error? If so, we could raise the more generic `UWError` instead, or define a new subclass to describe the actual problem. I'm not saying there IS an opportunity for change here, just suggesting that it's something to keep in mind.

But I'm fine with what's here (probably minus the `ABC`), as it at least lays the groundwork for more specific error types.